app/web/src/index.css

```diff
@@ -32,19 +32,6 @@
 
 }
 
-@layer components {
```

Comment:
Don't know why these didn't work.

Not sure I 100% love the pattern anyway since these styles are are very specific to one component and now completely seperate. I to optimize for delete-ability, and this would make it tricky to clean up `app/web/src/components/IdeSideBar/IdeSideBar.tsx` should we delete it down the track.

Do you think it would be better to add these classes into a new css file in `app/web/src/components/IdeSideBar/IdeSideBar.css` and import that directly into `app/web/src/components/IdeSideBar/IdeSideBar.tsx` instead @franknoirot?

Reply:
Yeah that's a good point, I think you're right having it local to the components directory is best.